src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas2/Cas2ReportsTest.kt

```diff
@@ -141,29 +141,29 @@ class Cas2ReportsTest : IntegrationTestBase() {
         withCreatedByUser(user2)
         withCrn("CRN_2")
         withNomsNumber("NOMS_2")
-        withCreatedAt(newerCreated.atOffset(ZoneOffset.ofHoursMinutes(0, 0)))
+        withCreatedAt(newerCreated)
         withData("{}")
-        withSubmittedAt(newerSubmitted.atOffset(ZoneOffset.ofHoursMinutes(0, 0)))
+        withSubmittedAt(newerSubmitted)
       }
 
       // outside time limit -- should not feature in report
       cas2ApplicationEntityFactory.produceAndPersist {
         withId(applicationId3)
         withApplicationSchema(applicationSchema)
         withCreatedByUser(user2)
-        withCreatedAt(tooOldCreated.atOffset(ZoneOffset.ofHoursMinutes(0, 0)))
+        withCreatedAt(tooOldCreated)
         withData("{}")
-        withSubmittedAt(tooOldSubmitted.atOffset(ZoneOffset.ofHoursMinutes(0, 0)))
+        withSubmittedAt(tooOldSubmitted)
       }
 
       val event1Details = Cas2ApplicationSubmittedEventDetailsFactory()
-        .withSubmittedAt(oldSubmitted)
+        .withSubmittedAt(oldSubmitted.toInstant())
```

Comment:
If the Postgres takes the time with zone and converts it to a utc time, do we also need to do this before we pass it in? [I figured that's what we were doing above](https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/1701/files#diff-5321e95ac99d42dcb5127d2aaffd665804f3aff8da93b397ca4bb75783ec7e07R133).